### ISSUE TITLE
Specific theme in shuffle plays default music instead of bgm 

### ISSUE THREAD

**Issue by Somedude12345540** [astronautlevel2/Anemone3DS#334]
I installed around 4 or 5 themes on shuffle, and for some reason only one of them plays default bgm

To reproduce this issue
1. Queue the themes for shuffle “including one
2. Restart the system
3. Switch to the broken theme

I expect the music to play as well, as it does play when I install it singularly, but will not play the themes bgm when installed in shuffle

New 3DS

11.17.0.50U

Anemone v3.0.1

Luma3DS v13.1.2

**Comment by astronautlevel2** [comment]
Can you upload the themes you are testing with and specify which theme doesn't play audio when shuffled?

**Comment by cooolgamer** [comment]
Isn't it a theme that diesn't have BGM enabled? Anemone fix this while installing, maybe it doesn't while installing in shuffle

**Comment by astronautlevel2** [comment]
That's correct - correcting the BGM enabled flag is intended primarily to be used with doing a BGM only install on top of a theme that doesn't have BGM enabled, something which isn't possible when doing a shuffle install. I'd consider checking it in shuffle install also but I'd want to measure the performance impact of it first.

I also want to (separately from the above) verify that that's what's happening here, so I'd need OP to tell us which theme isn't getting audio.

**Comment by cooolgamer** [comment]
Isn't it easier to just fix the flag if a bgm is found by downloading on the themeplaza browser?

Or/and correct this flag while uploading themes on themeplaza for new themes

**Comment by Somedude12345540** [comment]
> Can you upload the themes you are testing with and specify which theme doesn't play audio when shuffled?

The theme is on theme plaza named "Bee and Puppy Cat"
